src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -153,7 +153,7 @@ def update_data(self):
                 ipn = ipaddress.ip_network(ipnstr)
                 ent = Namespace.dbs_get_all(self.db_conn, mibs.APPL_DB, routestr, blocking=False)
                 if ent:
-                    nexthops = ent["nexthop"]
+                    nexthops = ent.get("nexthop", "")
```

Comment:
If we return default empty string, the below for loop will get executed once.
```
>>> foo = ""
>>> for bar in foo.split(','):
...   print("hello")
... 
hello
```
Can we default to None and add additional check to execute the for loop only if not None.